src/controllers/controller.bar.js

```diff
@@ -365,8 +372,10 @@ module.exports = DatasetController.extend({
 
 		helpers.canvas.clipArea(chart.ctx, chart.chartArea);
 
+		// float-bar support, if y arguments are array function will use bottom value as bar start point
```

Comment:
this comment seems like it could probably be removed. it doesn't seem like it really corresponds to the code below

Reply:
Ok! removed